Makefile

```diff
@@ -94,10 +94,6 @@ ldflags += $(LDFLAGS)
 ldflags := $(strip $(ldflags))
 
 BUILD_FLAGS := -tags "$(build_tags)" -ldflags '$(ldflags)'
-# check for nostrip option
-ifeq (,$(findstring nostrip,$(COSMOS_BUILD_OPTIONS)))
-  BUILD_FLAGS += -trimpath
```

Comment:
I think trimpath is for reproducible build?

Reply:
yeah.. this part is the one I wanted to discuss as I had put on my comment. Having the `trimpath` breaks the reflection module we are using for openRPC but it does help for reproducible builds

Reply:
@yihuang @crypto-facs is this currently blocking this PR from getting merged?